cmd/verify/verify.go

```diff
@@ -33,33 +35,38 @@ var (
 
 func NewCommand() *cobra.Command {
 	var clientConfig clientcmd.ClientConfig
-	var kubeconfigPath string
 	cmd := &cobra.Command{
 		Use:               "verify",
 		Short:             "Verify if arlon cli can run",
 		Long:              "Verify if required kubectl,argocd,git access is present before profiles and bundles are created",
 		DisableAutoGenTag: true,
-		Example:           "arlonctl verify --kubeconfigPath",
+		Example:           "arlonctl verify",
 		RunE: func(c *cobra.Command, args []string) error {
-			err := verify(clientConfig, kubeconfigPath)
+			err := verify(clientConfig)
 			if err != nil {
 				return err
 			}
 			return nil
 		},
 	}
 	clientConfig = cli.AddKubectlFlagsToCmd(cmd)
-	cmd.Flags().StringVar(&kubeconfigPath, "kubeconfigPath", "", "kubeconfig file location")
 	return cmd
 }
 
-func verify(clientConfig clientcmd.ClientConfig, kubeconfigPath string) error {
+func verify(clientConfig clientcmd.ClientConfig) error {
 	var err error
 	config, err := clientConfig.ClientConfig()
 	if err != nil {
 		return fmt.Errorf("failed to get k8s client config")
 	}
 
+	kubeconfigPath := os.Getenv("KUBECONFIG")
+	if kubeconfigPath == "" {
+		kubeconfigPath = filepath.Join(
```

Comment:
We can use default discovery rules instead of passing in ~/.kube/config as a parameter when creating a kube client.

Reply:
Done